### DISCUSSION TITLE
NetworkX 3.1

### DISCUSSION THREAD

**Discussion by rgreen13** [alibaba/GraphScope#3272] [General]
It seems the current version of GraphScope uses an interface like NetworkX 2.8. Are there plans to adjust some of these interfaces to 3.1? I'm particularly interested in the bounded simple_cycles functionality.

**Comment by yecol** [comment]
Hi @rgreen13, 

Thanks for your suggestion! We will check the compatibility and upgrade it soon!
Stay tuned~

/cc @acezen
